codecs/oxipng/src/lib.rs

```diff
@@ -20,6 +24,23 @@ pub fn optimise(
     } else {
         Interlacing::None
     });
+    match deflater {
```

Comment:
Would be nice to introduce a type here first, [like we have in the `resize` module](https://github.com/GoogleChromeLabs/squoosh/blob/19beb1a7ab5ab7df9625edaf7c3bf71a50e183ae/codecs/resize/src/lib.rs#L71-L77), so we can match on semantically meaningful words rather than numbers.

Also, invalid numbers should be `panic()`’d, not just use a default value.

Reply:
Thanks for the review @surma !

Is this what you were thinking of e3bb3a946cd06cc491b4682ac02fa80f6dc00c7f ?